src/sas/sasgui/guiframe/gui_manager.py

```diff
@@ -2527,7 +2529,10 @@ def show_data1d(self, data, name):
         if data.dy is not None:
             text += 'dY_min = %s:  dY_max = %s\n' % (min(data.dy), max(data.dy))
         text += '\nData Points:\n'
-        x_st = "X"
+        text += "<index> \t<X> \t<Y> \t<dY> \t<dX"
+        if data.dxl is not None and data.dxw is not None:
+                text += "l> \t<dxw"
+        text += ">\n"
         for index in range(len(data.x)):
```

Comment:
Yes, it works but splitting a tag in the middle is kind of magical.

How about:
```
text += "<index> \t<X> \t<Y> \t<Y>"
text += " \t<dX>" if data.dxl is None or data.dxw is None else " \t<dXl> \t<dXw>"
```

Note: changed `dxw` to `dXw` for consistency.